core/events_inc.php

```diff
@@ -135,6 +135,7 @@
 	'EVENT_BUGNOTE_DELETED' => EVENT_TYPE_EXECUTE,
 	'EVENT_TAG_ATTACHED' => EVENT_TYPE_EXECUTE,
 	'EVENT_TAG_DETACHED' => EVENT_TYPE_EXECUTE,
+	'EVENT_BUG_FILE_ATTACH' => EVENT_TYPE_EXECUTE,
```

Comment:
1. Rename to `EVENT_FILE_ADDED` 
2. Add a comment `# File events` above it.
3. While you are at it, please also add `# Tag Events` about the tag attached/detached.

Reply:
At the expense of "Tag Events" did not immediately understand. Now add!

Reply:
I also renamed according to the proposed format.